great_tables/_utils_render_html.py

```diff
@@ -717,10 +717,11 @@ def _get_table_defs(data: GTData) -> dict[str, Any]:
     # Get the table's width (which or may not have been set)
     table_width = data._options.table_width.value
 
-    # Get all the widths for the columns as a list where None values mean that the width is
-    # not set for that column
-    # TODO: ensure that the stub column is set first in the list
-    widths = data._boxhead._get_column_widths()
+    # Get all the widths for the columns (this includes the stub column)
+    boxh_final = data._boxhead.final_columns(options=data._options)
```

Comment:
Can we return a list of columns for the result of `final_columns`? It seems likely this boxhead with columns dropped (rather than hidden) violates some assumption of our data model.

Reply:
Now done. Thanks!